### DISCUSSION TITLE
Make it possible to add assignees to pull requests

### DISCUSSION THREAD

**Discussion by AndersRobstad** [gitbutlerapp/gitbutler#5424] [Ideas]
It would be very helpful to be able to add reviewers to the PR's that are created through the Gitbutler client. Right now I have to manually open GitHub everytime I create a PR through the client to be able to add the relevant reviewers. Thus, making it so that its not possible to eliminate the need for opening Github to create new PRs

**Comment by Byron** [comment]
Thanks for the suggestion! It makes total sense to aim for a more complete workflow, which I'd hope is possible through the GitHub API. CC @ndom91 who I think has had a fair share of GH API exposure.
